mapto-rest/src/test/java/org/uniprot/api/mapto/controller/AbstractMapToControllerIT.java

```diff
@@ -161,7 +161,7 @@ void submitJobToFinishAndVerifyGetDetails() throws Exception {
                 .andExpect(jsonPath("$.from", equalTo("UNIPROTKB")))
                 .andExpect(jsonPath("$.to", equalTo("UNIREF")))
                 .andExpect(jsonPath("$.query", equalTo(query)))
-                .andExpect(jsonPath("$.includeIsoform", equalTo("false")));
+                .andExpect(jsonPath("$.includeIsoform", equalTo(false)));
     }
 
```

Comment:
When i am running test, i can see a redis testcontainer being started. Why do we need redis container, we don't depend on it?